src/components/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import { stub } from "sinon";
+
+import Radio from "./Radio";
+
+describe("Radio Button", () => {
+  let container;
+  let changeHandler;
+  let clickHander;
+
+  before(() => {
+    clickHander = stub();
+    changeHandler = stub();
+    container = Enzyme.mount(
+      <Radio
+        id="inputID"
+        attributes={{ onClick: clickHander }}
+        onChange={changeHandler}
+      ></Radio>
+    );
+  });
+
+  it("Renders without crashing", () => {
+    expect(container.find("input").exists()).to.equal(true);
+  });
+
+  it("The radio element is an input with type='radio'", () => {
+    expect(container.find("input").prop("type")).to.equal("radio");
+  });
+
+  it("Allows user to pass in additional attributes", () => {
+    container.simulate("click");
+    expect(clickHander.callCount).to.equal(1);
+  });
+
+  it("Changing the value calls the onChange handler", () => {
+    container.find("input").simulate("change", { target: { value: "Hello" } });
+    expect(changeHandler.callCount).to.equal(1);
+  });
+
+  it("Renders with label", () => {
```

Comment:
Can you add to this test that the `htmlFor` (`htmlfor` in case the react version doesn't get picked up by Enzyme) is the same as the `id` that is passed as a prop to `Radio`? Having a label is great, but unless it's associated to the `input` element then it's not useful.

Reply:
Done!